tests/factories.py

```diff
@@ -63,9 +63,9 @@ class Meta:
     project_description = fuzzy.FuzzyText(length=20)
     project_slug = fuzzy.FuzzyText(length=20)
     project_start = fuzzy.FuzzyNaiveDateTime(datetime(2006, 1, 1))
-    project_end = fuzzy.FuzzyNaiveDateTime(datetime.now(), datetime(2020, 1, 1))
+    project_end = fuzzy.FuzzyNaiveDateTime(datetime(2020, 1, 1))
     nomination_start = fuzzy.FuzzyNaiveDateTime(datetime(2006, 1, 1))
-    nomination_end = fuzzy.FuzzyNaiveDateTime(datetime.now(), datetime(2020, 1, 1))
+    nomination_end = fuzzy.FuzzyNaiveDateTime(datetime(2020, 1, 1))
```

Comment:
Rather than change both the factory and the test, since the failure was that the [`start_dt` was no longer earlier than the `end_dt`](https://factoryboy.readthedocs.io/en/latest/fuzzy.html#fuzzynaivedatetime), what do you think about changing these to `fuzzy.FuzzyNaiveDateTime(datetime.now(), datetime.now()+relativedelta(years=5))` and leaving the test unchanged? That would keep the `project_end` and `nomination_end` being future dates meaning the `ProjectFactory` would retain an "active" state.

Reply:
Yea..I can do that.